hugo/content/3-queries/operators.md

```diff
@@ -212,7 +218,7 @@ For the following code samples, assume that a User has a field `scores`, which c
     ```
 
 - `{{% a_blank "$elemMatch" "https://docs.mongodb.com/manual/reference/operator/elemMatch/" %}}`
-
+    <a name="elemMatch"></a>
```

Comment:
This creates an anchor, so that you can link to it from the same page. At the top of the page there's a part that says "...see the Mangrove documentation on __elemMatch__ ..." and there's a link to this part of the documentation. 

@adamchel do you know a better way to do this in Markdown / Hugo? AFAIK Hugo only provides anchors for headings (created using `#`), so for list items you have to add these anchors manually.